src/repr/src/row/encoding.rs

```diff
@@ -998,4 +1002,34 @@ mod tests {
             Ok(())
         );
     }
+
+    fn scalar_type_columnar_roundtrip(scalar_type: ScalarType) {
+        use mz_persist_types::parquet::validate_roundtrip;
+        let mut rows = Vec::new();
+        for datum in scalar_type.interesting_datums() {
```

Comment:
Such a convenient method to exist...

Reply:
i didn't check, but pretty likely this was matt. we had something like this at crl, and it was nice there, too :)